### PR TITLE
Add `Table` model, save all table references to `metadata.tables`, re…

### LINKED ISSUE #50: Only latest table name is saved
Sql queries can have several schemas/table names.

```python
>>> stmt = sqlparse("""SELECT a FROM sys.t JOIN doc.b ON t.a = b.a""")[0]
>>> print(stmt.metadata)
Metadata(schema='doc', table_name= 'b', parameterized_properties={}, with_properties={})
```

`sys.t` table information is lost, it's easy to fix, the question is about the resulting API

`Metadata(schema=['sys', 'doc'], table_name=['t', 'b'], parameterized_properties={}, with_properties={})` would this be good enough, where schemas and table_name are ordered?

If no schema it'd be: 
`Metadata(schema=[None, 'doc'], table_name=['t', 'b'], parameterized_properties={}, with_properties={})`

Now that you are integrating into this, wdyt? @amotl

### PR DESCRIPTION
**Pull request by surister**
…move `table_name` and `schema` from metadata.

Fixes #50 

## Checklist

 - [x] Link to issue this PR refers to (if applicable): 
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by surister** [comment]
Apparently `return f'{quote(self.schema) + '.' if self.schema else ""}{quote(self.name)}'` is not valid in Python 3.8, don't even remember why, will check it tomorrow. Still good to review.

File: cratedb_sqlparse_py/cratedb_sqlparse/models.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+
+@dataclasses.dataclass
+class Table:
+    name: str
+    schema: str = None
+
+    @property
+    def fqn(self):
+        return f'{quote(self.schema) + '.' if self.schema else ""}{quote(self.name)}'
```
Comment (amotl) [inline cratedb_sqlparse_py/cratedb_sqlparse/models.py:15]:
```suggestion
        return quote(self.schema) + ("." if self.schema else "") + quote(self.name)
```

File: cratedb_sqlparse_py/cratedb_sqlparse/parser.py [hunk 2]
```diff
@@ -132,8 +132,7 @@ class Metadata:
     table, schema, columns, options...
     """
 
-    schema: str = None
-    table_name: str = None
+    tables: list = dataclasses.field(default_factory=list)
```
Comment (amotl) [inline cratedb_sqlparse_py/cratedb_sqlparse/parser.py:135]: Suggestion to use `typing.List`, in order to stay compatible with Python 3.8.
```suggestion
    tables: List[Table] = dataclasses.field(default_factory=list)
```

**Review by amotl — APPROVED** [review]
Excellent, thanks!

**Pull request merged**